src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -440,7 +440,7 @@ number of days to wait for starting the program stage.
     for a repeatable stage after you click *Complete* for an event
     of the stage in data entry form. This field is selected by
     default.
-    5. Enter **Standard interval days**. The number of days to repeat the repeatable program stage.  
+    5. Enter **Standard interval days**. When scheduling an event in a repeatable stage, and there are previous events in that stage, this fields value will be the suggested interval between the latest event in the repetable stage and the scheduled date of the new event.
```

Comment:
I think this can be simplified a bit.
```suggestion
    5. Enter **Standard interval days**. This value will be the suggested interval between the last event in a repeatable stage and the scheduled date of the next event.
```